home-api-tests/pom.xml

```diff
@@ -13,4 +13,223 @@
+                                            <time>10000</time>
+                                        </wait>
+                                    </run>
+                                </image>
+                                <image>
+                                    <name>${project.artifactId}</name>
+                                    <build>
+                                        <from>${docker.image.jdk.version}</from>
+                                        <entryPoint>
+                                            <exec>
+                                                <args>java</args>
+                                                <args>-jar</args>
+                                                <args>-D${datasource.url.property.name}=${postgres.url}</args>
+                                                <args>${home.application.jar.name}-${home.application.jar.version}.jar
+                                                </args>
+                                            </exec>
+                                        </entryPoint>
+                                        <assembly>
+                                            <basedir>/</basedir>
+                                            <inline>
+                                                <files>
+                                                    <file>
+                                                        <source>
+                                                            ${docker.artifacts.path}/${home.application.jar.name}-${home.application.jar.version}.jar
+                                                        </source>
+                                                        <outputDirectory>/</outputDirectory>
+                                                    </file>
+                                                </files>
+                                            </inline>
+                                        </assembly>
+                                    </build>
+                                    <run>
+                                        <ports>
+                                            <port>${home.application.external.port}:${home.application.internal.port}
+                                            </port>
+                                        </ports>
+                                        <links>
+                                            <link>${postgres.container.name}</link>
+                                        </links>
+                                        <wait>
+                                            <log>Started HomeApplication</log>
+                                            <time>30000</time>
+                                        </wait>
+                                    </run>
+                                </image>
+                            </images>
+                        </configuration>
+                        <executions>
+                            <execution>
```

Comment:
What is about to check if the containers with the same name as we are going to run are running and stop them before?

Reply:
Done